### ISSUE TITLE
One-sample t_test() as a test of proportions

### ISSUE THREAD

**Issue by LKobilke** [joon-e/tidycomm#46]
Currently, the package supports one-sample t-tests with a specified mean value (mu). However, with master students, we frequently work with percentages (i.e. proportions) as opposed to raw means. We might want to update that option.

**Comment by MarHai** [comment]
Good idea. We will have more updates when it comes to getting the package ready for the master students so I've created a respective label that does not require fixing for the christmas commit I'd say.
